rewatch/src/build/packages.rs

```diff
@@ -19,6 +19,7 @@ use std::time::SystemTime;
 #[derive(Debug, Clone)]
 pub struct SourceFileMeta {
     pub modified: SystemTime,
+    pub is_type_dev: bool,
```

Comment:
I don't think it's relevant to put here, I would put it on the Module struct

Reply:
We would still need it here if we want to put it in Module struct.
`packages::parse_packages` will loop over all the source_files and by that time, we no longer know what "type" they had.